hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationFuture.java

```diff
@@ -144,8 +144,9 @@ public void run() {
      * 'set' calls are ignored.
      *
      * @param offeredResponse The type of response to offer.
+     * @return <tt>true</tt> is offered response is set, <tt>false</tt> otherwise.
```

Comment:
New version:
> @return <tt>true</tt> if offered response, either a final response or an internal response, is set/applied, <tt>false</tt> otherwise. If <tt>false</tt> is returned, that means offered response is ignored because a final response is already set to this future.